tests/admin.test.js

```diff
@@ -16,6 +18,9 @@ const password = 'password23423';
 const adminUrl = '/api/admin/users';
 
 describe('Users', () => {
+  before(() => {
```

Comment:
@nkpremices which of the tests in this file requires an email to be sent to the user?

Reply:
No one @MemunaHaruna 
I added this line in this file because it is executed the first and also because the stub that will be created here will remain until the end of the mocha process